components/namespace-lister/base/nginx.conf

```diff
@@ -0,0 +1,59 @@
+worker_processes auto;
+error_log /var/log/nginx/error.log;
+pid /run/nginx.pid;
+
+# Load dynamic modules. See /usr/share/doc/nginx/README.dynamic.
+include /usr/share/nginx/modules/*.conf;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    log_format upstreamlog '[$time_local] $remote_addr - $remote_user - $server_name $host to: $proxy_host  $upstream_addr: $request $status upstream_response_time $upstream_response_time msec $msec request_time $request_time';
```

Comment:
Minor: This is logging a lot of info. We can simplify it. I'd suggest using the same format we have in `ui`

Reply:
This is the same as what we're using in `components/ui/base/proxy/nginx.conf`.